.evergreen/run-tests.sh

```diff
@@ -120,7 +120,12 @@ if [ "$SLOW_TESTS_ONLY" == "true" ]; then
     ./gradlew -PjdkHome=/opt/java/${JDK} -Dorg.mongodb.test.uri=${MONGODB_URI} \
               ${TRANSACTION_URI} ${GRADLE_EXTRA_VARS} ${ASYNC_TYPE} --stacktrace --info testSlowOnly
 else
+    set +x
```

Comment:
I see a lot of docs for `set -x` but none for `set +x`.  Can you point me to something explaining what it does/tell me why you're doing this?

Reply:
Sure NP! 😸  `+` turns off an option and `-` turns on an option.  Thus `set -x` would result in a trace of all the commands and their arguments being printed and `set +x` would disable this behavior. 

Source: https://ss64.com/bash/set.html
Relevant sections:
> Using + rather than - will cause the option to be turned off. 


and

>    -x  Print a trace of simple commands and their arguments
       after they are expanded and before they are executed